web/src/plugins/airtable.js

```diff
@@ -47,6 +47,25 @@ Airtable.install = function (Vue) {
       });
     }
   };
+
+  Vue.prototype.$speakerTopics = new Map();
+
+  Vue.prototype.$getSpeakerTopics = function (speakerID, successCallback, errorCallback) {
+    if (this.$speakerTopics.hasKey(speakerID)) {
+      // return cached response
+      successCallback(this.$speakerTopics.get(speakerID));
+    } else {
+      this.$db('Topics').select({ view: 'All', fields: ['name'], filterByFormula: "({People} = 'TODO')" }).firstPage((error, records) => {
```

Comment:
I wasn't finding a contains / in operator in the filterByFormula docs. https://support.airtable.com/hc/en-us/articles/203255215-Formula-Field-Reference#array_functions

Reply:
If you follow the approach I mentioned on the comments above, then you don't need this whole function :)